### ISSUE TITLE
ARIA attributes must conform to valid values in the facility form state selector on error

### ISSUE THREAD

**Issue by emyl3** [CDCgov/prime-simplereport#4329]
## Area or Component
SimpleReport app
Settings > Manage Facilities > Facility form > Facility section > State selector on error
![Screen Shot 2022-09-27 at 11.16.41.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/f7d149ad-2a47-49b0-9364-061c235cab8e)

## Quick Summary
When there is a validation error for the selected state the original `<select>` element for the state input has an `aria-describedby" value that does not exist as an `id` value.

### How to Replicate
- Go to Settings
- Click "Manage facilities"
- Either edit an existing facility or add a new one
- Select a State from the dropdown in the Facility form section that is not supported by SimpleReport (e.g. DC)
- Press "Tab" key
- Run the axeDevTools check and you will see the violation

## Accessibility Violation
ARIA attributes must conform to valid values

## Action Requested
Make sure an `id` value same as the `aria-describedby` is correctly attributed to a `.usa-error-message` `<span>` for the state selector error like the other error messages in that form.

## Screenshots (optional)
![Screen Shot 2022-09-27 at 11.27.08.png](https://images.zenhubusercontent.com/5ffc8b9b4950f50bf5829d88/f8c5e80c-a65d-4418-bd1e-2c407460310a)

## Source Code (optional)

[which file or line is the issue attached to?]: #

## Original Deque Issue
https://dequeuniversity.com/rules/axe/4.4/aria-valid-attr-value?application=AxeChrome
https://cdc-devtools.dequecloud.com/issues/e19a6044-3fdd-4532-a185-402f6c84c05a
